third_party/optimized-svg-icons/styles.css

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
we're going to probably need to add a LICENSE file in this folder as well as a README.amp that contains any modification. see https://github.com/ampproject/amphtml/blob/master/third_party/mustache/README.amp.

you might want to commit the original code **and then** through a pR do the modification in another PR (or just another commit) so we can see a clear delta of what has changed.

/cc @cramforce